.travis.yml

```diff
@@ -1,3 +1,6 @@
+sudo: required
```

Comment:
Am I right that `sudo: required` is necessary in order to use trusty? If so, a comment to that effect would be good; hopefully that will change at some point in the future, and then we can stop requiring sudo again.